src/thing/thing.ts

```diff
@@ -115,7 +115,7 @@ export function getThing(
 }
 
 /**
- * Get all [[Thing]]s about which a [[SolidDataset]] contains Quads.
+ * Get all [[Thing]]s in a [[SolidDataset]].
```

Comment:
Perhaps we should also link to https://docs.inrupt.com/developer-tools/javascript/client-libraries/structured-data/ ?